fuzz/fuzz_targets/coprocessor/codec/mysql/decimal.rs

```diff
@@ -11,7 +11,7 @@ where
     I: Iterator<Item = u8>,
 {
     let mut bytes = [0u8; 8];
-    for byte in bytes.iter_mut() {
+    for byte in &mut bytes {
```

Comment:
em, why include this in this PR?

Reply:
this PR also makes clippy covers all workspaces (introduced in previous commits), and this is an issue that clippy reports.